File: service/src/main/java/org/cadadr/conjugaison/repository/VerbConjugationRepository.java
package org.cadadr.conjugaison.repository;

import org.cadadr.conjugaison.domain.VerbConjugation;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface VerbConjugationRepository extends MongoRepository<VerbConjugation, String> {

    VerbConjugation findByInfinitif(String infinitif);

}

File: service/src/main/kotlin/org/cadadr/conjugaison/repository/VerbConjugationRepository.kt
package org.cadadr.conjugaison.repository

import org.bson.types.ObjectId
import org.cadadr.conjugaison.domain.VerbConjugation
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.data.mongodb.repository.Query

interface VerbConjugationRepository : MongoRepository<VerbConjugation, ObjectId> {

    @Query(value = "{}", fields = "{ 'infinitif' : 1 }")
    fun infinitif(): List<VerbConjugation>

}
